script/munin-node-configure

```diff
@@ -30,6 +30,7 @@ my $config = Munin::Node::Config->instance();
 my @all_families     = qw/auto manual contrib/;
 my @default_families = qw/auto/;
 
+$ENV{PATH} = '/sbin:/bin:/usr/sbin:/usr/bin';
```

Comment:
This reduced (or extended?) PATH would be used by all plugins run via `munin-node-configure`, or?
This could be a surprise, if the local admin wants to make non-packaged installed programs accessible to plugins?